websites/D/Discords.com/presence.ts

```diff
@@ -195,14 +195,136 @@ presence.on("UpdateData", async () => {
         url: document.location.href
       }
     ];
+    // discord.bio
+  } else if (document.location.pathname.includes("/profiles")) {
+    presenceData.largeImageKey = "discordbio_logo";
+    presenceData.smallImageKey = "reading";
+    presenceData.smallImageText = "Browsing...";
+    presenceData.details = "Browsing top bios";
+    presenceData.buttons = [
+      {
+        label: "View Bios",
```

Comment:
i don't think it's correct

Reply:
In what terms? Code or like grammatically?

Reply:
grammatically

Reply:
<img src="https://cdn.tixte.com/uploads/cdn.anaxes.xyz/kpkbdu8009a.png"/>

I was confused with that at the start too, 'cause I had bio's.

Reply:
english is weird